Tribler/Core/permid.py

```diff
@@ -47,3 +48,24 @@ def save_pub_key(keypair, pubkeyfilename):
     with open(pubkeyfilename, 'w') as file:
         file.write(membuf.read())
     membuf.close()
+
+
+def generate_keypair_multichain():
+    return LibNaCLSK()
```

Comment:
Why are we using `LibNaCL` and not `M2Crypto` to generate this key? Also, is it possible to let `dispersy.crypto` generate the key (like in https://github.com/Tribler/tribler/blob/devel/Tribler/Core/APIImplementation/LaunchManyCore.py#L242)?